config/locales/en.yml

```diff
@@ -333,6 +333,7 @@ en:
         meta_description: Meta Description
         front_page: Front Page
         active: Active
+        public: Public Category Content
```

Comment:
I actually was thinking about this more- do you think we should actually have three states- internal only, public only, and both?  In this case it would work to have two checkboxes (or three radio buttons).  What do you think?